### PR TITLE
Iswg 1234 up

### PR DESCRIPTION
**Pull request by ChallyDeanPFG**
Dean bringing pull to dev for first time

### CODE REVIEW

File: commitExercise.txt [hunk 1]
```diff
@@ -1 +1,7 @@
-
+<<<<<<< HEAD
```
Comment (KristenRunyan) [inline commitExercise.txt]: please fix your merge problem

File: commitExercise.txt [hunk 2]
```diff
@@ -1 +1,7 @@
...
+Local Change line 1 on iswc-1234
+Local Change line 2 on iswc-1234
+=======
+local change line 1 on iswg-1234
+local change line 2 on iswg-1234
+>>>>>>> 1e63e50914aba1f005f02326f690cfd979adf442
```
Comment (gartee-john-PFG) [inline commitExercise.txt]: looks like an incomplete merge

**Review by gartee-john-PFG — CHANGES_REQUESTED** [review]

**Review by KristenRunyan — CHANGES_REQUESTED** [review]
comment

File: commitExercise.txt (hunk 1 above)
Comment (ChallyDeanPFG) [reply commitExercise.txt]: Updated how does it look now

**Review by gartee-john-PFG — APPROVED** [review]
looks good

**Review by KristenRunyan — APPROVED** [review]

**Pull request merged**